actionkit/api/order.py

```diff
@@ -1,8 +1,45 @@
 from actionkit.api.base import ActionKitAPI
 
-
 class AKOrderAPI(ActionKitAPI):
 
+    def get_order_detail(self, order_id):
```

Comment:
Noting that we seem to have diverging naming conventions here. User and event both use the shorter `get_user` and `get_event` while order and recurring order add `_detail` to the function names.

Reply:
That's an intentional divergence. The basic order or recurring order record isn't all that useful by itself, so get_order_detail makes an additional call to grab the billing information from the linked order_detail record. I can add the basic get_ methods if we think they're useful. In the context of what we're doing with Ganymede, it's useful mainly to 1) get basic info on all of a person's orders with the list method, and then 2) get more info on an individual order, which is what get_order_detail does.